### ISSUE TITLE
brocken salt-dependency during yum update

### ISSUE THREAD

**Issue by 4815162342lost** [saltstack/salt#47268]
Hello, i tried to upgrade my system via yum update, but receive dependency error from salt-repository:

```
--> Processing Dependency: libsodium.so.13()(64bit) for package: zeromq-4.1.4-6.el7.x86_64
--> Finished Dependency Resolution
Error: Package: zeromq-4.1.4-6.el7.x86_64 (@saltstack-repo)
           Requires: libsodium.so.13()(64bit)
           Removing: libsodium-1.0.5-1.el7.x86_64 (@saltstack-repo)
               libsodium.so.13()(64bit)
           Updated By: libsodium-1.0.16-1.el7.x86_64 (saltstack-repo)
              ~libsodium.so.23()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

```
 Salt Version:
           Salt: 2017.7.4

Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.5 (default, Aug  4 2017, 00:39:18)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: centos 7.4.1708 Core
         locale: ISO-8859-1
        machine: x86_64
        release: 3.10.0-693.17.1.el7.x86_64
         system: Linux
        version: CentOS Linux 7.4.1708 Core
```
```		
[root@hostname~]# rpm -qa | grep  libsodium
libsodium-1.0.5-1.el7.x86_64
```

Issue can be resolved by reinstalling **salt-minion** and **libsodium** package (yum remove : libsodium; yum install salt-minion).

But could you please check this case and provide feedback? I can not reinstall salt-minion on each server manually.

My repo for salt:
baseurl=https://repo.saltstack.com/yum/redhat/$releasever/$basearch/latest

Thanks in advance.

**Comment by gtmanfred** [comment]
@dmurphy18 can you take a look at this?

Thanks,
Daniel

**Comment by rnickle** [comment]
I can confirm that I'm having this problem with RHEL 7.4, been having it a few weeks, this is the first time I tried removing libsodium and salt-minion and reinstalling:

sudo yum remove salt-minion libsodium; sudo yum install salt-minion

Worked fine, thanks.  I have a few machines still suffering if you have an alternate approach to test.

**Comment by Pesticles** [comment]
This.

**Comment by gtmanfred** [comment]
This is a duplicate of saltstack/salt-pack#531

**Comment by dmurphy18** [comment]
The has been resolved in the next point release by bumping the release number for zeromq.
see https://github.com/saltstack/salt-pack/issues/531
